content/applications/inventory_and_mrp/purchase/products/pricelist.rst

```diff
@@ -0,0 +1,260 @@
+       `1` and `2`.
+     - N/A
+     - sequence
+   * - Company
+     - Name of company the product belongs to.
+     - :guilabel:`Company` field in the vendor pricelist.
+     - company_id
+   * - :ref:`External ID <purchase/products/external-id>`
+     - Unique ID of a record used to update existing vendor pricelists.
+     - N/A
+     - id
+
+Import records
+--------------
+
+With a template downloaded, fill out the XLSX or CSV file with the necessary information. After
+inputting everything, import the file back into the Odoo database, by going to
+:menuselection:`Purchase app --> Configuration --> Vendor Pricelists`.
+
+On the page, click the :icon:`fa-cog` :guilabel:`(gear)` icon in the top-left corner. In the
+drop-down menu that appears, click :guilabel:`Import records`.
+
+Then, click :guilabel:`Upload File` in the upper-left corner, and after selecting the XLSX or CSV
+file, confirm the correct fields, and click :guilabel:`Import`.
+
+.. seealso::
+   - :doc:`../../../essentials/export_import_data`
+   - :ref:`Common fields <purchase/products/common-fields>`
+
+.. image:: pricelist/supplier-pricelist-example.png
+   :align: center
+   :alt: Upload file screen.
+
+Formatting import file
+~~~~~~~~~~~~~~~~~~~~~~
+
+To understand how to format import files for vendor pricelists, consider the following example.
+
+- `Storage Box` (:guilabel:`Reference`: `E-COM08`) is sold by `Wood Corner` for `$10`.
+- `Large Desk` (:guilabel:`Reference`: `E-COM09`) has no records in the vendor pricelist.
+
+An import file is created to do the following:
+
+- Update the price for `Wood Corner` from `$10` to `$13`
+- Add pricelist for `Storage Box`: the vendor, `Ready Mat` intends to sell the product for `$14`
+- Add pricelist for `Large Desk`: vendor is `Wood Corner`, price is `$1299`
+- Add pricelist for `Large Desk`: vendor is `Azure Interior`, price is `$1399`.
+
+.. list-table:: Vendor pricelist data
+   :header-rows: 1
+
+   * - id
+     - company_id
+     - delay
+     - price
+     - product_tmpl_id
+     - sequence
+     - partner_id
+   * - product.product_supplierinfo_3
+     - My Company (San Francisco)
+     - 3
+     - 13.00
+     - [E-COM08] Storage Box
+     - 4
+     - Wood Corner
+   * -
+     - My Company (San Francisco)
+     - 3
+     - 14.00
+     - [E-COM08] Storage Box
+     - 5
+     - Ready Mat
+   * -
+     - My Company (San Francisco)
+     - 2
+     - 1299.00
+     - [E-COM09] Large Desk
+     - 6
+     - Wood Corner
+   * -
+     - My Company (San Francisco)
+     - 4
+     - 1399.00
+     - [E-COM09] Large Desk
+     - 7
+     - Azure Interior
+
+.. note::
+   The *Technical field name* was used to create this information.
+
+.. note::
+   To download the sample files for reference, click the following links:
+
+   - :download:`Download XLSX <pricelist/pricelist-example.xlsx>`
+   - :download:`Download CSV <pricelist/pricelist-example.csv>`
```

Comment:
Nitpick, up to you to implement– suggesting to reword to have more semantic links
```suggestion
   Download the sample files for reference:

   - :download:`Sample XLSX import file <pricelist/pricelist-example.xlsx>`
   - :download:`Sample CSV import file <pricelist/pricelist-example.csv>`
```